src/Utils.cpp

```diff
@@ -180,6 +178,32 @@ unsigned long CUtils::GetLongIP(const CString& sIP) {
     return ret;
 }
 
+CString CUtils::GetHostName() {
+    const char *pEnv;
+
+    pEnv = getenv("HOSTNAME");
+    if (pEnv && pEnv[0])
+        return pEnv;
+
+#if defined(HAVE_GETHOSTNAME) && defined(HAVE_SYSCONF)
+    long HostNameMax = sysconf(_SC_HOST_NAME_MAX);
```

Comment:
long int iHostNameMax

Reply:
How about `UnameBuffer`? That's a struct. `sUnameBuffer` or...?

Reply:
ZNC's naming convention is based on modified [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)

`s` is for strings. Since the scope of uname variable is so limited, whatever works, no prefix is fine